src/nvim/api/autocmd.c

```diff
@@ -812,41 +810,19 @@ void nvim_exec_autocmds(Object event, Dict(exec_autocmds) *opts, Error *err)
   api_free_array(patterns);
 }
 
-static bool check_autocmd_string_array(Array arr, char *k, Error *err)
```

Comment:
This was copy-pasted? Lift `check_string_array` into validate.c and use it instead.